vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -149,26 +149,22 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
      * ========================================================================================== */
 
     private lateinit var configurationViewModel: ConfigurationViewModel
-    private lateinit var sessionListener: SessionListener
-    protected lateinit var bugReporter: BugReporter
-    private lateinit var pinLocker: PinLocker
 
+    @Inject lateinit var sessionListener: SessionListener
```

Comment:
Some of the properties were kept private on purpose, we are losing this. I don't like having @Inject in Base Activity/Fragment class

Reply:
with @Inject in Views, nothing can be private... I will update for the `SessionListener` instance.